frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -385,7 +376,7 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                                     ...event,
                                     playerTime: eventPlayerTime,
                                     playerPosition: eventPlayerPosition,
-                                    isOutOfBand,
+                                    capturedInWindow: !!event.properties.$window_id,
```

Comment:
We don't actually use this anywhere but at some point we could indicate this fact in the UI